src/LieGroups.jl

```diff
@@ -1,38 +1,48 @@
 module LieGroups
 
-using ManifoldsBase, Manifolds
-
-# before removing them from Manifolds.jl – let's for consistency include them here but
-# overwrite them for now. This makes moving away from that (in Manifolds.jl 0.11) here nonbreaking.
-import Manifolds: apply, apply!
-
+using ManifoldsBase, Manifolds, LinearAlgebra
+
+#
+#
+# = Compatibility (and a bit of type piracy for now)
+# The following imports are necessary to use Manifolds.jl 0.10 with Lie Groups
+# The line is removed when the Groups are removed from possibly 0.11
+import Manifolds: apply, apply!, identity_element, is_identity, compose
+# Both define the following structs, so these for now lead to asking for explicit prefixes
+# Manifolds: Identity, TranslationGroup
 include("documentation_glossary.jl")
 include("interface.jl")
+include("Lie_algebra/Lie_algebra_interface.jl")
+# Generic Operations
+include("group_operations/addition.jl")
+
+# Actions
 include("group_actions/group_action_interface.jl")
 include("group_actions/group_operation_action.jl")
-include("additive_group_operation.jl")
 
-include("groups/additive.jl")
+# Lie groups
+include("groups/translation_group.jl")
 
 export LieGroup, LieAlgebra
 
 export AbstractGroupOperation, Identity
-export AdditiveGroupOperation
+export AdditionGroupOperation
 
 export AbstractGroupActionType, AbstractGroupAction
 export AbstractLeftGroupActionType, AbstractRightGroupActionType
 export LeftGroupOperation, RightGroupOperation
 export InverseLeftGroupOperation, InverseRightGroupOperation
 export GroupOperationAction
 
-export AdditiveGroup
+export TranslationGroup
 
-export apply, apply!, diff_apply, diff_apply!
-export base_manifold
-export adjoint,
-    adjoint!,
-    compose,
-    compose!,
+export adjoint, adjoint!, apply, apply!
+export base_Lie_group, base_manifold
```

Comment:
Can we stop using capital letters in function names? They make them look inconsistent.

Reply:
for `Lie` I was just not sure, since it is a last name, I would always capitalise that, for example also in adjectives. But I also do not have a strong opinion, just my personal taste would be to keep `_Lie_` in function names. If you really do not like that, we can surely rename that.

Reply:
Thanks. I prefer when function names don't use capital letters at all.

Reply:
Sure. I am just German, wE cApItAliZe a lot. But as long as it is consistent, I do not mind – hope to remember then (because German).